src/components/dashboard/PendingTxs/PendingTxListItem.tsx

```diff
@@ -35,7 +36,15 @@ const PendingTx = ({ transaction, url }: PendingTxType): ReactElement => {
           <Box gridArea="confirmations">
             {isMultisigExecutionInfo(transaction.executionInfo) ? (
               <Box className={css.confirmationsCount}>
-                {`${transaction.executionInfo.confirmationsSubmitted}/${transaction.executionInfo.confirmationsRequired}`}
+                <SvgIcon
+                  component={OwnersIcon}
+                  inheritViewBox
+                  fontSize="small"
+                  sx={{ color: ({ palette }) => palette.text.primary }}
```

Comment:
Can this be inside the icon itself?

Reply:
I tried but it overrides the other places where we use this icon.

In the queue the icon is either grey or black/green.
In the dashboard its always black/white.

Might be a good idea to make this more consistent but maybe there is a reason the icons + text are not green on the dashboard in dark mode. cc @liliiaorlenko